experiment/compatibility-versions/e2e-skip-version-testing.sh

```diff
@@ -66,6 +72,11 @@ run_skip_version_tests() {
   # Test removal of emulated version entirely.
   export PREV_RELEASE_BRANCH="release-${EMULATED_VERSION}"
   delete_emulation_version || ret=$?
+  if [[ "$EMULATED_VERSION" != "$initial_emulated_version" ]]; then
+    # overwrite node e2e tests with the node tests at the initial emulated version because Kubelet is still at that version.
+    echo "Replace ${EMULATED_VERSION} node tests with node tests at ${initial_emulated_version}"
+    rsync -av --delete "./test/e2e/node" "${TMP_DIR}/prev-release-k8s-${initial_emulated_version}/test/e2e/node"
```

Comment:
oh, I just saw this (sorry I've been out recently), I don't think this is safe, we can have an incompatible change in the e2e framework or ginkgo for example.

We should can filter out the `MinimumKubeletVersion:1.X` tests when skipping (and also revisit with SIG Node a better way to handle these, in the future NodeFeatureDiscovery should help)

Also, this didn't work for n-2:

https://testgrid.k8s.io/sig-api-machinery-emulated-version#skip-version-test-n-minus-2&width=20